Corona-Warn-App/src/test/java/de/rki/coronawarnapp/ccl/configuration/update/CclConfigurationUpdaterTest.kt

```diff
@@ -41,19 +43,21 @@ internal class CclConfigurationUpdaterTest : BaseTest() {
 
         coEvery { boosterRulesRepository.update() } returns UpdateResult.UPDATE
         coEvery { cclConfigurationRepository.updateCclConfiguration() } returns UpdateResult.NO_UPDATE
+        coEvery { dccValidationRepository.updateInvalidationRules() } returns UpdateResult.NO_UPDATE
```

Comment:
I think the function names in this Test Class should now also in include ` ... and invalidation rules...`. Currently, they are something like `updateIfRequired() should update booster rules and ccl configuration if required`()`